comp/core/workloadmeta/collectors/internal/containerd/image.go

```diff
@@ -244,7 +244,10 @@ func (c *collector) handleImageCreateOrUpdate(ctx context.Context, namespace str
 	return c.notifyEventForImage(ctx, namespace, img, bom)
 }
 
-func (c *collector) notifyEventForImage(ctx context.Context, namespace string, img containerd.Image, sbom *workloadmeta.SBOM) error {
+func (c *collector) createOrUpdateImageMetadata(ctx context.Context,
```

Comment:
could you add a description to this function.
for example explains in which case the first return value (`*workloadmeta.ContainerImageMetadata`) can be `nil`